File: desktop/src/utils/remote/command.py
from enum import Enum


class Command(Enum):
    SHOW_TEXT = 'show_text'
    HIDE_TEXT = 'hide_text'
    PLAY_VIDEO = 'play_video'
    STOP_VIDEO = 'stop_video'
    SHOW_HTML = 'show_html'
    HIDE_HTML = 'hide_html'
    SET_VOLUME = 'set_volume'

File: app/src/services/command.ts
import { api } from "../utils/api"

export namespace CommandService {
    export async function showText(text: string): Promise<void> {
        await api.post('/command/show_text', { text })
    }

    export async function hideText(): Promise<void> {
        await api.post('/command/hide_text')
    }

    export async function playVideo(params: { dirname: string, filename: string }): Promise<void> {
        await api.post('/command/play_video', params)
    }

    export async function stopVideo(): Promise<void> {
        await api.post('/command/stop_video')
    }

    export async function showHTML(html: string): Promise<void> {
        await api.post('/command/show_html', { html })
    }

    export async function hideHTML(): Promise<void> {
        await api.post('/command/hide_html')
    }
}